src/components/side_panel/chart/calendar_chart/calendar_chart_config_panel.ts

```diff
@@ -0,0 +1,38 @@
+import { _t } from "../../../../translation";
+import { Granularity } from "../../../../types";
+import { GenericChartConfigPanel } from "../building_blocks/generic_side_panel/config_panel";
+
+export class CalendarChartConfigPanel extends GenericChartConfigPanel {
+  static template = "o-spreadsheet-CalendarChartConfigPanel";
+
+  groupByChoices = [
+    { value: "year", label: _t("Year") },
+    { value: "month", label: _t("Month") },
```

Comment:
Could we use `month_number` instead of `month` ? 
With that, we could have a consistency with pivots, and you could use `ALL_PERIODS` filtered.
WDYT ?